Automattic/pocket-casts-android
# New commands in Tasker Action "Control Playback": "Skip Forward" and "Skip Back"

## Description
Adds ability for the **Control Playback** action in Tasker to skip the number of provided seconds either forward or backward


## Testing Instructions
- Open Tasker
- Add **Pocket Casts Control Playback** action
- Set **Playback Command** to **Skip Forward**
- Set **Time To Skip** to **600**
- Press the **OK** button
- Start playing a podcast in Pocket Casts
- Run the Tasker action and check that it skips 10 minutes forward

## Screenshots or Screencast 
![screencapture (16)](https://user-images.githubusercontent.com/4864031/198047423-cad42a00-cf52-48fb-be6b-f608b95a46c8.png)


## Checklist
 
#### I have tested any UI changes...
